sdc/crypto/encrypter.py

```diff
@@ -2,19 +2,20 @@
 from sdc.crypto.jwt_helper import JWTHelper
 
 
-def encrypt(json, key_store, key_purpose):
+def encrypt(json, key_store, key_purpose, encryption_service=None):
```

Comment:
As discussed, this sounds a bit like it's the service doing the encryption. Possibly `downstream_service`, `target_service`, `encryption_for_service` or even `service` might be more appropriate. We can update the docstring below to qualify whatever we decide on.

Reply:
Decided to go with `encryption_for_service`.

Reply:
Decided to go with `encryption_for_service`.